package.json

```diff
@@ -153,6 +153,14 @@
       }
     ],
     "commands": [
+      {
+        "command": "powerpages.powerPagesFileExplorer.powerPagesRuntimePreview",
```

Comment:
Do we wish to show these commands in command palette also?

Reply:
No for now - updating accordingly